Create usage documentation

buchholz-sean opened the pull request:
Documentation for Max Bot usage once installed. Includes a link in README.md to the usage doc.

buchholz-sean committed:
Create usage documentation

Documentation for Max Bot usage once installed. Includes a link in README.md to the usage doc.

buchholz-sean commented on usage.md:
```diff
@@ -0,0 +1,72 @@
+Removes all roles from the user. _Tabula rasa_, baby.
+
+
+## Administrative commands
```
Note that the broadcast functionality hasn't been added to a release yet. Everything from line 58 down can be scrapped or edited as necessary.

MROwens reviewed (approved):
Everything looks good to me!

ghost commented:
would look into getting some documentation for nodemailer seeing as we are using there services to setup the email verification, possible option could be:

## nodemailer

```npm install nodemailer --save```
Max-Bot uses [nodemailer](https://nodemailer.com/about/) to send verification emails to students that are attempting to verify their Full Sail University email address.

buchholz-sean commented:
Great call, @joegreen2. Will update.

buchholz-sean committed:
Update README.md

Added information about Nodemailer module. Also replaced a reference to `.env` file with `max.config.js` which is current correct env config file.
